packages/client/hmi-client/src/components/workflow/tera-workflow.vue

```diff
@@ -299,7 +299,7 @@ const saveNodeStateHandler = debounce(async () => {
 	const updatedWorkflow = await workflowService.updateState(wf.value.getId(), nodeStateMap);
 	nodeStateMap.clear();
 	wf.value.update(updatedWorkflow, false);
-}, 300);
+}, 250);
```

Comment:
Whats the thought process behind changing this? 
asking out of curiosity more than concern

Reply:
Makes sense, thank you